packages/blade/src/components/Input/OTPInput/OTPInput.stories.tsx

```diff
@@ -25,6 +25,12 @@ export default {
     onChange: ({ name, value }): void => {
       console.log(`input field ${name} content changed to ${value}`);
     },
+    onBlur: ({ name, value, inputIndex }): void => {
```

Comment:
<img width="662" alt="image" src="https://user-images.githubusercontent.com/30949385/225518950-e7f6cbb9-1f94-451a-be65-b90fc311334c.png">

It is showing up outside of category.


Also I am not able to see console log when I focus on OTPInput in storybook 🤔

Reply:
You have to see it in the action tab
<img width="625" alt="Screenshot 2023-03-16 at 12 06 47 PM" src="https://user-images.githubusercontent.com/24487274/225534788-24d84a43-ad74-4a13-9174-cc3ba36f426b.png">

Reply:
ooo TIL about actions tab lol. I was checking in browser console

Reply:
storybook redirects all `on*` console logs to the actions tab 😅